aces/Loris
# removed support for dynamic user generated visit labels

This removes the possibility to enter a custom visit label dynamically when creating a new timepoint. This is to lay the foundation of normalised visits and help the move of visitLabels from the config.xml to the database.

collaterals: the suggestion options that was custom to user generated visitlabels was removed, Deprecation error is thrown id config.xml not updated before going on this branch

reqs: removing the `generation`, `regex`, `length` and `suggest` tags from the config.xml under the `visitLabel` section.

To test:
- first, before checking out this branch, set the generation to user and confirm that you can enter the visit label manually.
- checkout the branch and make sure an error is thrown when config is not modified
- change the generation=user to generation = sequence, this should not throw an error but we still recommend to completely remove the tags (this is just to simplify the upgrade for projects already using sequence)
- completely remove the tags mentioned above and go through the testplan.md file
